File: samples/command/src/perl/sample.pl
# This file is a "Hello, world!" in Perl language for wandbox.
use FindBin;
use lib "$FindBin::Bin/subdir";

BEGIN {
  unshift @INC, "$FindBin::Bin/";
}

require test0;
require './test1.pl'; require test2;

print "Hello, Wandbox!\n";
test0();
test1();
test2();

# Perl language references:
#   http://perldoc.perl.org/

File: samples/command/src/r/sample.R
# This file is a "Hello, world!" in R language for wandbox.

data(iris)                               ## the "hello, world" of statistics.
summary(iris)                            ## a simple summary of the columns
fit <- lm(Sepal.Length ~ . , data=iris)  ## basic regression
summary(fit)                             ## summary

cat("All done\n")

source("test1.R"); source("test2.R")

test1()
test2()

# R language references:
#   https://cran.r-project.org/manuals.html
